### PR TITLE
1.11 Backport: repeat the stream

### PR DESCRIPTION
**Pull request by nLight**
Backport https://github.com/dcos/dcos-ui/pull/2832

### CODE REVIEW

**Comment by nLight** [comment]
run integration tests

**Comment by juliangieseke** [comment]
adding high prio because its 1.11 👍 
@nLight can you please update it? 🙇

**Comment by DanielMSchmidt** [comment]
run integration tests

**Comment by nLight** [comment]
run integration tests

**Comment by juliangieseke** [comment]
run integration tests

**Comment by juliangieseke** [comment]
run all tests

**Comment by Poltergeist** [comment]
run Integration Tests

**Comment by nLight** [comment]
run Integration Tests

File: src/js/core/MesosStream.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { stream } from "@dcos/mesos-client";
+import { ReplaySubject } from "rxjs/ReplaySubject";
 
 export const MesosStreamType = Symbol("MesosStreamType");
 export default stream({ type: "SUBSCRIBE" }, "/mesos/api/v1?subscribe")
-  .publishReplay()
+  .repeat()
+  .multicast(() => new ReplaySubject())
```
Comment (Fabs) [inline src/js/core/MesosStream.js]: This is the same as publishReplay:

https://github.com/Reactive-Extensions/RxJS/blob/master/src/modular/observable/replay.js

**Comment by nLight** [comment]
Checking those integration fails locally

**Comment by nLight** [comment]
run system tests

**Comment by nLight** [comment]
run integration tests

**Comment by ahoskins** [comment]
run integration tests

**Comment by juliangieseke** [comment]
❌ 😢

**Comment by nLight** [comment]
run integration tests

**Comment by nLight** [comment]
run system tests

**Comment by nLight** [comment]
run integration tests

**Comment by juliangieseke** [comment]
~relieve is one (two) reviews away :trollface: https://github.com/dcos/dcos-ui/pull/2838~ it is `hold merge` :(

**Comment by nLight** [comment]
run integration tests

**Comment by juliangieseke** [comment]
merged in #2862

**Pull request closed without merging**